app/models/edition.rb

```diff
@@ -187,4 +187,8 @@ def editors
     user_ids = statuses.pluck(:created_by_id) + revisions.pluck(:created_by_id)
     User.where(id: user_ids.uniq)
   end
+
+  def assets
+    image_revisions.flat_map(&:assets) + file_attachment_revisions.flat_map(&:assets)
+  end
```

Comment:
This is probably more of a concern of a revision than an edition. I'd suggest putting it there and delegating on edition.

Reply:
Sure!